CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 ### Added
+- Initialisation - Allow asynchronous middlewares (ie. Liana.init).
```

Comment:
Let's wait for the `forest-express-sequelize` and `forest-express-mongoose` to consider this PR.
Once liana v6 released this PR will need liana version update in `writePackageJson` function.